zerver/tests/test_message_edit.py

```diff
@@ -1328,7 +1328,7 @@ def notify(user_id: int) -> Dict[str, Any]:
 
         # This code path adds 9 (1 + 4/user with muted topics) + 1 to
         # the number of database queries for moving a topic.
-        with self.assert_database_query_count(19):
+        with self.assert_database_query_count(21):
```

Comment:
Why does this add 2 database queries -- is that the `bulk_create` having been more efficient?

If we need these queries, we need to update the comment, but I think it'd be better to avoid adding more queries if at all possible.

Reply:
> is that the bulk_create having been more efficient?

Yes.

After reverting back to `bulk_create`, all the extra database queries added up in the previous commit are now reverted.